### DISCUSSION TITLE
Support repos like Remo which has a mix-product release log

### DISCUSSION THREAD

**Discussion by nikolay** [aquaproj/aqua#2183] [Deprecated (Feature Request)]
### Feature Overview

Please, take a look at the release of Rome: https://github.com/rome/tools/releases

### Why is the feature needed?

Tools like Rome would be suitable for JSON formatting, but I've seen other projects having a release type.

**Comment by suzuki-shunsuke** [comment]
I'm not familiar with Rome at all, but I think aqua can support Rome.

https://docs.rome.tools/standalone-executable/#install-rome

> To install Rome, grab the executable for your platform from the [latest CLI release](https://github.com/rome/tools/releases) on GitHub and give it execution permission.

The tag prefix is `cli/`.

I created a pull request to add rome to aqua-registry.

- https://github.com/aquaproj/aqua-registry/pull/14815

**Comment by suzuki-shunsuke** [comment]
> Tools like Rome would be suitable for JSON formatting, but I've seen other projects having a release type.

Could you explain more detail?
